.circleci/config.yml

```diff
@@ -49,6 +49,9 @@ parameters:
   pre-publish:
     type: boolean
     default: false
+  scheduled-build:
```

Comment:
TODO - set this value automatically to match whatever is in the package.json. I don't think we will ever have a time where this value will be different.

Reply:
Redoing this piece - just realized this isn't needed.